test/server/cards/01_SOR/events/RallyingCry.spec.ts

```diff
@@ -0,0 +1,44 @@
+describe('Rallying Cry', function () {
+    integration(function (contextRef) {
+        describe('Rallying Cry\'s ability', function () {
+            beforeEach(function () {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        hand: ['rallying-cry'],
+                        groundArena: ['death-trooper'],
+                        spaceArena: ['imperial-interceptor']
+                    },
+                    player2: {
+                        groundArena: ['rugged-survivors'],
```

Comment:
Seems like we can remove rugged survivors for this test, it's not being used for anything. Be aware that will change the Death Trooper test since it will just automatically select the base as the only target, so the attack will happen with just one click.